### PR TITLE
Added spanish translation of autoclass_tutorial.mdx

### PR DESCRIPTION
**Pull request by duedme**
# Translation of autoclass_tutorial.mdx into spanish

I made the translation of autoclass_tutorial.mdx into Spanish (fixes https://github.com/huggingface/transformers/issues/15947). The document is located in the transformers/docs/source_es folder.

@omarespejel 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

### CODE REVIEW

File: docs/source_es/autoclass_tutorial.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Recuerda, la arquitectura se refiere al esqueleto del modelo y los checkpoints son los pesos para una arquitectura dada. Por ejemplo, [BERT](https://huggingface.co/bert-base-uncased) es una arquitectura, mientras que `bert-base-uncased` es un checkpoint. Modelo es un término general que puede significar una arquitectura o un checkpoint.
+
+</Tip>
+
+En este tutorial, aprende a:
+
+* Cargar un tokenizador preentrenado.
+* Carga un extractor de características preentrenado.
+* Carga un procesador preentrenado.
+* Carga un modelo preentrenado.
```
Comment (omarespejel) [inline docs/source_es/autoclass_tutorial.mdx:28]:
```suggestion
* Cargar un modelo preentrenado.
```

**Review by omarespejel — CHANGES_REQUESTED** [review]
Thank you! Some nits.

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
Thank you @Duedme! 🤗

@sgugger LGTM 👍

**Comment by sgugger** [comment]
There are a lot of conflicts in your branch, for just one added file. I think the easiest to fix this is to copy your file, create a new fresh branch from the main branch and open a new PR, as I can't review the diff here :-)

**Pull request closed without merging**